mlflow/pytorch/__init__.py

```diff
@@ -79,6 +79,7 @@ def log_model(
     await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS,
     requirements_file=None,
     extra_files=None,
+    save_as_state_dict=False,
```

Comment:
Can we avoid introducing this boolean parameter and provide a separate method (e.g. `log_state_dict`) isntead?

Reply:
created new method `log_state_dict`